source/support/update-govwifi-server-certificate.html.erb

```diff
@@ -27,11 +31,52 @@ description: Updating the GovWifi server certificate to keep using the service
+            select Continue.
+          </li>
           <li>Your device will automatically switch to the new certificate and reconnect to GovWifi.</li>
           <li>You can continue using GovWifi as normal.</li>
         </ol>
 
+        <h2 class="govuk-heading-m">Troubleshoot</h2>
+
+        <h3 class="govuk-heading-s">You are unable to connect to GovWifi</h3>
+        <p class="govuk-body">If you are unable to connect to GovWifi, try re-entering your GovWifi username and
+          password. In some cases, you may also need to re-boot your device or rejoin your GovWifi network</p>
+        <p class="govuk-body">If you are still experiencing problems, contact your IT support desk.</p>
+
+        <h3 class="govuk-heading-s">You get an error that the Thumbprint/Fingerprint is incorrect</h3>
+        <p class="govuk-body">A Thumbprint (also sometimes referred as a Fingerprint) is an ID that is uniquely
+          calculated from a certificate. It is used to identify the certificate, as a validation check, or as a way to
+          manage certificates in devices. Since a new certificate will have a different fingerprint, some users may
+          get a confusing message that the Thumbprint/Fingerprint is incorrect.</p>
+        <p class="govuk-body">If you get this message, follow these steps:
+        <p>
+        <ol class="govuk-list govuk-list--number">
+          <li>Validate the thumbprint for the new certificate matches one of (as of 11 November 2019):
+            <ul class="govuk-list govuk-list--bullet">
+              <li>SHA1 Fingerprint=37:CE:B5:7A:3F:38:EB:7C:66:A2:4E:90:BB:11:71:D7:BC:4D:65:1A</li>
+              <li>SHA256
+                Fingerprint=58:33:C1:B7:34:35:95:94:EA:18:AF:9A:58:71:B2:E9:44:B1:07:08:57:E8:EA:A2:A2:BE:73:D8:CE:5C:DD:B7
+              </li>
+            </ul>
+          </li>
+          <li>Accept the new certificate</li>
+          <li>You can continue using GovWifi as normal</li>
+        </ol>
+
+        <h3 class="govuk-heading-s">You don't have permissions to trust the GovWifi server certificate</h3>
+        <p class="govuk-body">If you are using a device managed by your organisation, you may have restricted
+          permissions and may be unable to trust the new GovWifi certificate, the intermediate certificate, or the
+          root certificate. Contact your IT support desk that should be able to accept them for you.</p>
+        <p class="govuk-body">In some cases, your IT support desk may need access to the new certificates. They can be
+          downloaded here:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li><%= link_to "GovWifi certificate", "/assets/wifi.service.gov.uk.crt", class: "govuk-link" %></li>
+          <li><%= link_to "Intermediate certificate", "/assets/GeoTrustRSACA2018.crt", class: "govuk-link" %></li>
+          <li><%= link_to "Root certificate", "/assets/DigiCertGlobalRootCA.crt", class: "govuk-link" %></li>
```

Comment:
These paths don't seem to be valid - may be you're planning on adding these files and I'm just getting ahead of myself! 🤷‍♂

Reply:
The new certs are here:
> GovWifi certificate: https://docs.wifi.service.gov.uk/assets/wifi.service.gov.uk.crt 
> Intermediate certificate: https://docs.wifi.service.gov.uk/assets/GeoTrustRSACA2018.crt
> Root certification: https://docs.wifi.service.gov.uk/assets/DigiCertGlobalRootCA.crt

though I don't know what process is populating them